docs/logging.md

```diff
@@ -20,12 +20,19 @@ There are also many other utilities, command line and graphical, that can be
 used to view logcat logs from a connected android device in a more flexible
 manner.
 
-#### Changing the loglevel in Fenix
+#### Changing the loglevel in Firefox Android
 
 If you need more verbose logging, after the call to `RustLog.enable()` in
 `FenixApplication`, you may call `RustLog.setMaxLevel(Log.Priority.DEBUG,
 true)`.
 
-### Accessing logs when running iOS
+### Accessing logs when running Firefox iOS
+
+* Navigate to the settings page
+* tap the version number 5 times to get to the secret menu
+* Select "copy log files to container"
+* Find the logs on your files folder
+
```

Comment:
Also when using XCode you can view the logs in the debugger but that's kinda implied probably for people developing that way.